Task02-PixelPerfect/characters.css

```diff
@@ -0,0 +1,47 @@
+/**
+ * All Rights Reserved
+ * This software is property information of AKUREY S.A.
+ * Contact us at contact@akurey.com
+ * @summary short description for the file
+ */
+.text-item {
+  font-family: "Roboto Slab";
+  font-style: normal;
+  font-weight: 700;
+  font-size: 22px;
+  line-height: 28px;
+  color: #FFF;
+}
+
+.grid-container {
+  display: flex;
+  gap: 20px;
+}
+
+.grid-item {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  width: 219.74px;
```

Comment:
From now on, we are going to avoid the decimals number, except if it is strictly needed, but for most cases, you can put an integer number, you will notice the same result, cleaner and easier-to-read code.